.travis.yml

```diff
@@ -30,16 +30,14 @@ install:
 script:
     - sh travis.sh run_tests --coverage
     - sh travis.sh run_syntax_tests
+    - pycodestyle --statistics --count
 
 after_success:
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
             brew update;
             brew install python3;
-            pip3 install python-coveralls;
-      fi
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-            pip install python-coveralls;
       fi
+    - pip3 install python-coveralls pycodestyle;
```

Comment:
you need to install `pycodestyle` before running

Reply:
move it to install section then ?

Reply:
ya